src/site/includes/breadcrumbs.drupal.liquid

```diff
@@ -12,7 +12,7 @@
 
     {% for crumb in crumbs %}
       {% if forloop.last == true %}
-        <li><a onclick="recordEvent({ 'event': 'nav-breadcrumb', 'breadcrumb-click-label': '{{ crumb.text }}', 'breadcrumb-click-level': '{{ forloop.index }}', 'breadcrumb-total-levels': '{{ crumbs.length }}' });" aria-current="page" href="/{{ path }}">{{ crumb.text }}</a></li>
+        <li><a class="breadcrumb-link" onclick="recordEvent({ 'event': 'nav-breadcrumb', 'breadcrumb-click-label': '{{ crumb.text }}', 'breadcrumb-click-level': '{{ forloop.index }}', 'breadcrumb-total-levels': '{{ crumbs.length }}' });" aria-current="page" href="/{{ path }}">{{ crumb.text }}</a></li>
```

Comment:
If you want to split this up to a few lines it might make it a bit more readable for future devs!

```liquid
<li>
  <a
    aria-current="page"
    class="breadcrumb-link"
    href="/{{ path }}"
    onclick="recordEvent({ 'event': 'nav-breadcrumb', 'breadcrumb-click-label': '{{ crumb.text }}', 'breadcrumb-click-level': '{{ forloop.index }}', 'breadcrumb-total-levels': '{{ crumbs.length }}' });"
  >
    {{ crumb.text }}
  </a>
</li>
```